ios/brave-ios/Sources/BraveWallet/Crypto/Transaction Confirmations/TransactionStatusView.swift

```diff
@@ -317,82 +364,108 @@ struct TransactionStatusView: View {
       .foregroundColor(Color(braveSystemName: .textPrimary))
       .multilineTextAlignment(.center)
     case .confirmed:
-      // Sending
-      //
-      // Sent!
-      // 0.1 ETH has been sent to account Oxabcd
-      //
-      // Approving ERC20
-      //
-      // Approved!
-      // 1 DAI has been approved on your Account 1
-      //
-      // Swapping on Eth
-      //
-      // Completed!
-      // 2.3 USDC has been added to your Account 1
-      // on Ethereum Mainnet
-      //
-      // Swapping on Sol
-      //
-      // Completed!
-      // Solana Swap
-      // on Solana Mainnet
-      VStack {
-        if isTxSolSwap {
-          Text("\(Strings.Wallet.txStatusCompleted)!")
-            .font(.title2.weight(.semibold))
-            .padding(.bottom, 8)
-          HStack {
-            Text(Strings.Wallet.transactionSummarySolanaSwap)
-              .fontWeight(.semibold)
-            explorerButton
-          }
-          Text("\(Strings.Wallet.txStatusOn) \(activeTxNetwork?.chainName ?? "")")
-            .foregroundColor(Color(braveSystemName: .textSecondary))
-        } else if isTxEthSwap {
-          Text("\(Strings.Wallet.txStatusCompleted)!")
+      if case .cancel(let parsedTx) = txStatusStore.followUpAction {
+        // Transaction cancellation
+        //
+        // Transaction cancelled!
+        // 0.1 ETH has been reverted to account 0xf333...095F
+        //
+        VStack {
+          Text(Strings.Wallet.confirmedTransactionCancellationTitle)
             .font(.title2.weight(.semibold))
             .padding(.bottom, 8)
-          Text(txToValue)
+          Text(revertFromValue(parsedTx: parsedTx))
             .fontWeight(.semibold)
-            + Text(" \(Strings.Wallet.txStatusSwappedMsg)")
+            + Text(" \(txCancelledMsg(parsedTx: parsedTx)) ")
             .foregroundColor(Color(braveSystemName: .textSecondary))
           HStack {
-            Text(txStatusStore.activeParsedTx.fromAccountInfo.name)
+            Text("\(Strings.Wallet.confirmedTransactionCancellationAccount) ")
+              .foregroundColor(Color(braveSystemName: .textSecondary))
+              + Text(parsedTx.fromAccountInfo.address.truncatedAddress)
               .fontWeight(.semibold)
             explorerButton
           }
```

Comment:
Maybe designs don't permit this given the different font weight & colour, but should this be one localized string for translations? Not sure if other languages would change order of the values.
ex. `String.localizedStringWithFormat("%d %@ has been reverted to %2", value, symbol, address)`